### PR TITLE
Feature: Responsive UI updates

### PR DESCRIPTION
**Pull request by thomasdigby**
- Made a simple responsive menu/nav
- Added next and previous links in the footer of pages
- Fixed the white-space value on code examples

Screen Shot 2021-03-12 at 13 02 43
Screen Shot 2021-03-12 at 13 02 34

### CODE REVIEW

File: src/components/Navigation.tsx [hunk 1]
```diff
@@ -67,53 +168,45 @@ const SourceList: React.FC<SourceListProps> = ({ items }) => (
...
+      <StyledNavigation ref={ref} open={menuOpen}>
+        <Heading size="sm" css={{ mb: '$4' }}>
+          Atom Learning
+          <br />
+          Design System
+        </Heading>
+        {Object.entries(items).map(([source, content]) => (
+          <React.Fragment key={source}>
+            <SourceHeading>{capitalCase(source)}</SourceHeading>
+            {Array.isArray(content) ? (
```
Comment (hollg) [inline src/components/Navigation.tsx:189]: This section of the markup is pretty noisy and I found the ternary difficult to parse. Could the section in the ternary be pulled into a separate component, which could then use two return statements to improve readability?

File: package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "prism-react-renderer": "^1.1.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-feather": "^2.0.9",
```
Comment (andoulla) [inline package.json:35]: Does this mean that the `documentation` project will not be using the icons from the DS project?

Comment (thomasdigby) [reply package.json:35]: This is just temporary until we sort out the icons in the component lib, there are likely to be a fair few changes

Comment (andoulla) [reply package.json:35]: oh ok then.

**Review by andoulla — APPROVED** [review]

**Pull request merged**